scripts/check_doc_links.py

```diff
@@ -56,6 +56,7 @@
     "https://gateway.autonolas.tech/ipfs/<hash>,",  # non link (400)
     "https://github.com/valory-xyz/open-autonomy/trunk/infrastructure",  # svn link (404)
     "http://host.docker.internal:8545",  # internal (ERR_NAME_NOT_RESOLVED)
+    "https://notes.ethereum.org/@vbuterin/eip-1559-faq",  # 503 Service Temporarily Unavailable
```

Comment:
TODO: revert.

This PR will be continued later this week when autonomy test command allows capturing of logs at INFO level
https://github.com/valory-xyz/open-autonomy/blob/adea7fd23e3fef23674ae24776d095ff42579e2c/pytest.ini#L3